_docs/events/updated-transaction.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+section: events
+title: "Updated Transaction"
+order: 3
+---
+
+The `Updated Transaction` event should be pushed to the `digitalData.events` array when transaction data changed, e.g. user pay the bill, order delivered, canceled etc.
```

Comment:
when transaction data changed, e.g. user pay the bill, order delivered, canceled etc. ->
when transaction data has changed, e.g. user paid the bill, order was delivered/canceled etc.